### ISSUE TITLE
What is E in Chapter 9 - Adaptive Time Stepping?

### ISSUE THREAD

**Issue by bradcarman** [SciML/SciMLBook#137]
I'm assuming $\text{E}$ from [here](https://github.com/SciML/SciMLBook/blob/3091889761a3e6c9d90fb30d42e7e6dc9b1192a7/_weave/lecture09/stiff_odes.jmd#L596) is the residual function value, i.e. $g(u_i) = \text{E}$.  Is this correct?  And is it scalarized or in vector form?

Does the relative tolerance $\tau_r$ ever take into account the scaling of $u$?  For example, a hydraulic problem that has pressure values in the order of 1e7 and valve displacements in the order of 1e-6.  If $\text{E}$ is in vector form then is $q$ computed as `norm( E ./ (tau_r*u + tau_a) )` ?

**Comment by ChrisRackauckas** [comment]
No, E is an error estimate between two different methods. It's method-dependent, but generally the difference between two different stepping techniques at different orders.

>  If  is in vector form then is computed as norm( E ./ (tau_r*u + tau_a) ) ?

Yes https://docs.sciml.ai/DiffEqDocs/stable/extras/timestepping/

I'm going to write some notes for the new course that develop this a lot more.
